.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+#### Code Sample, a copy-pastable example if possible
+
+A "Minimal, Complete and Verifiable Example" will make it much easier for maintainers to help you:
+<http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports>
+
+```python
+# Your code here
+```
+
+#### Problem description
+
+This should explain **why** the current behaviour is a problem and why the expected output is a better solution.
+
+#### Expected Output
+
+#### Output of `xr.show_versions()`
```

Comment:
Keep this for now, but we need to make it easy to access the `climpred` version and ask for users to add that to the issue as well. I can handle this in a small PR.